omop_file_validator.py

```diff
@@ -176,6 +180,32 @@ def find_blank_lines(f):
     return [i + 1 for i in indices]
 
 
+def find_scientific_notation_errors(f, int_columns):
+    df = pd.read_csv(f, dtype=str)
+    df = df.rename(columns=str.lower)
+    df = df[[col for col in int_columns if col in df.columns]]
+
+    errors = []
+    sci_not_count = collections.defaultdict(int)
+
+    for submission_col_name in df.columns:
+        submission_column = df[submission_col_name]
+        for i, value in submission_column.items():
+            if pd.notnull(value) and re.match(SCIENTIFIC_NOTATION_REGEX,
+                                              value):
+                sci_not_count[submission_col_name] += 1
```

Comment:
It would be helpful to catch the full list of indices here instead of just the count

Reply:
Sure, should I just put the first index of an error in case the entire column has scientific notation?

Reply:
Yes, that should suffice. However it needs to be emphasized in the message.

Reply:
Sounds good, thanks.

Reply:
I made a change. How does it look now?